vueui/src/components/view/records/general.vue

```diff
@@ -10,7 +10,7 @@
       <vui-group-item label="Age:"><view-records-field :editable="(editable & 1) > 0" path="active.age"/></vui-group-item>
       <vui-group-item label="Sex:"><view-records-field :editable="(editable & 1) > 0" path="active.sex"/></vui-group-item>
       <vui-group-item label="Rank:"><view-records-field :editable="(editable & 1) > 0" path="active.rank"/></vui-group-item>
-      <vui-group-item label="Phisical Status:">
+      <vui-group-item label="Physical Status:">
         <view-records-field :editable="(editable & 1) > 0" path="active.phisical_status">
           <select v-model="$root.$data.state.editingvalue">
             <option v-for="i in choices.phisical_status" :key="i" :value="i">{{ i }}</option>
```

Comment:
Well, maybe fix actually variable name?

Reply:
Well, I wouldn't want to screw the whole thing up but sure. I can fix the entire active.phisical_status. Lemme take a look.